go/master/service.go

```diff
@@ -123,6 +214,8 @@ func (s *Service) GetTask(dummy int, task *Task) error {
 		return err
 	}
 
+	*task = t.Task
+
```

Comment:
@Yancey1989 I have print log mentioning how many times the task failed, but have not changed the type. Do you think if it's sufficient? :)